docker-compose.yaml

```diff
@@ -14,6 +14,49 @@ services:
             POSTGRES_PASSWORD: ${DATALAYER_POSTGRES_PASSWORD}
         networks:
             - datalayer
+        healthcheck:
+            test: ["CMD-SHELL", "pg_isready -U ${DATALAYER_PG_USER}"]
+            interval: 5s
+            timeout: 5s
+            retries: 5
+
+    api-hasura:
```

Comment:
lets pick more descriptive namings here 

instead of `api-hasura` => `datalayer-api`

instead of `graphql-engine` => `indexer-api`

instead of `envio-postgres` => `indexer-db` 

instead of `datalayer-postgres` => `datalayer-db` 

lmk wdyt : )

Reply:
agree on making them more descriptive but i like the tech being on the name

datalayer-graphql-api
indexer-postgres-db

but we can go with your naming too, don't have any particular preference